src/main/java/com/prgrms/vouchermanagement/voucher/domain/PercentDiscountVoucher.java

```diff
@@ -0,0 +1,33 @@
+package com.prgrms.vouchermanagement.voucher.domain;
+
+import java.util.UUID;
+
+public class PercentDiscountVoucher implements Voucher {
+	private final UUID voucherId;
+	private final long percent;
+
+	public PercentDiscountVoucher(UUID voucherId, long percent) {
+		if (percent <= 0 || percent > 100)
+			throw new IllegalArgumentException("PercentDiscountVoucher - percent should be between 0 and 100");
+
+		this.voucherId = voucherId;
+		this.percent = percent;
+	}
+
+	@Override
+	public UUID getVoucherId() {
+		return voucherId;
+	}
+
+	@Override
+	public long discount(long beforeDiscount) {
+		return beforeDiscount * (percent / 100);
```

Comment:
```suggestion
		return beforeDiscount * (100 - percent / 100);
```

이거 계산식 확인해봐야 하지 않을까요!?